src/Features/Core/Portable/FindUsages/DefinitionItem.DocumentLocationDefinitionItem.cs

```diff
@@ -26,9 +26,10 @@ internal sealed class DefaultDefinitionItem : DefinitionItem
                 ImmutableArray<TaggedText> originationParts,
                 ImmutableArray<DocumentSpan> sourceSpans,
                 ImmutableDictionary<string, string> properties,
+                ImmutableArray<AdditionalProperty> displayableProperties,
```

Comment:
it's an array of 'AdditionalProperty'.  so 'AdditionalProperties' is probably a better name.  Also, consider just calling this a "FindUsagesProperty" as opposed to "Additional".